package.json

```diff
@@ -19,6 +19,7 @@
     "build-android": "node scripts/inject.js --platform android",
     "build-windows": "node scripts/inject.js --platform windows",
     "build-integration": "node scripts/inject.js --platform integration",
+    "bundle-trackers": "node scripts/bundleTrackers.mjs > build/tracker-lookup.json",
```

Comment:
I don't think this works on windows - just do the write inside the `bundleTrackers.mjs` to fix

Reply:
I was trying to avoid having different code paths for each platform (extension uses this). I'll think of some way to resolve.